terraform/projects/pr-dashboard/network.tf

```diff
@@ -20,6 +20,15 @@ variable "subnet_cidr_blocks" {
   ]
 }
 
+variable "subnet_wpt_blocks" {
```

Comment:
You can get this from the module with  `${module.subnet.ids}`, as exported here:
https://github.com/bocoup/infrastructure-web-platform/blob/master/terraform/modules/aws/subnet/main.tf#L9

My apologies if I sent you down the wrong path with the earlier suggestions.

Reply:
Also, to see the attributes that become available with each terraform resource you create, check the "attribute reference" in the docs:
https://www.terraform.io/docs/providers/aws/r/subnet.html#attributes-reference

....or just look at the `.tfstate` file directly, which is a giant json file under `.terraform` in the project folder.